### PR TITLE
Cosmetic update

### PR DESCRIPTION
**Pull request by magikcypress**
I worked for a cosmetic code with a new helpers.
Clean for test CI
Update PrivateBin version 1.1

### CODE REVIEW

File: scripts/restore [hunk 1]
```diff
@@ -55,14 +49,16 @@ sudo chown -R root:root $final_path
...
-# Restore conf files
-sudo cp -a "./conf/nginx.conf" $conf
-
-# Reload Nginx
-sudo service nginx reload
+# Restore configuration files
+sudo cp -a ./conf/nginx.conf "${nginx_conf}"
 
 # Set ssowat config
+ynh_app_setting_set "$app" is_public "$is_public"
```
Comment (M5oul) [inline scripts/restore]: already set, why do you set it again?

File: scripts/restore [hunk 2]
```diff
@@ -55,14 +49,16 @@ sudo chown -R root:root $final_path
...
 if [ "$is_public" = "Yes" ];
 then
-  sudo yunohost app setting $app unprotected_uris -v "/"
+  ynh_app_setting_set "$app" unprotected_uris "/"
 fi
+
+# Reload service
+sudo systemctl reload nginx
+sudo yunohost app ssowatconf
```
Comment (M5oul) [inline scripts/restore]: not needed.
Must be already done by YunoHost after this the execution of this script.

**Comment by maniackcrudelis** [comment]
All seems ok for me too.
Except points shown by Moul, I agree with him.

But, it needs to be test with an upgrade from the current version.

File: scripts/restore (hunk 1 above)
Comment (magikcypress) [inline scripts/restore]: in fact is_public is yes is not relevant ? so prefer is_public is no ?
```
# Set ssowat config
if [ "$is_public" = "No" ];
then
	ynh_app_setting_delete $app skipped_uris
fi
```

Thanks

**Comment by JimboJoe** [comment]
Seems that all existing pastes are removed/made  unavailable during upgrade process.
Maybe it is by design?
If that's the case, there should be a warning in the README (and in the forum post later).

**Pull request merged**